Snowplow/SPUtilities.m

```diff
@@ -25,6 +25,7 @@
 #import "SPPayload.h"
 #import "SPSelfDescribingJson.h"
 #import "SPScreenState.h"
+#include <sys/sysctl.h>
```

Comment:
You are now double importing this for the `#elif SNOWPLOW_TARGET_OSX` block.  If you remove that you should now have the correct imports.

Reply:
Oh, somehow I missed that, will remove, thanks!